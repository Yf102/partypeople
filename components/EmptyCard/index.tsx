import React from 'react'
import styles from '../PartnerCard/PartnerCard.module.css'
import cn from 'classnames'

const EmptyCard = () => {
  return (
    <div
      data-testid='empty-card-element'
      className={cn(
        styles['light'],
        'col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 my-4'
      )}
    >
      <h1 className='text-xl'>
        We could not find any partner less then 100km away!
      </h1>
    </div>
  )
}

export default EmptyCard
