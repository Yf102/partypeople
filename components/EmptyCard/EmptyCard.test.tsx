import React from 'react'
import { render, screen } from 'tests/test-utils'
import EmptyCard from 'components/EmptyCard/index'

describe('EmptyCard', () => {
  it('It renders EmptyCard', async () => {
    const { asFragment } = render(<EmptyCard />)
    expect(screen.getByTestId('empty-card-element')).toBeTruthy()
    expect(asFragment()).toMatchSnapshot()
  })
})
