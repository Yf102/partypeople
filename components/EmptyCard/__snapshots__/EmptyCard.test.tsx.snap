// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyCard It renders EmptyCard 1`] = `
<DocumentFragment>
  <div
    class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 my-4"
    data-testid="empty-card-element"
  >
    <h1
      class="text-xl"
    >
      We could not find any partner less then 100km away!
    </h1>
  </div>
</DocumentFragment>
`;
